<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRecipe</class>
 <widget class="QDialog" name="NewRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="2">
    <widget class="QCheckBox" name="veganBox">
     <property name="text">
      <string>Vegan</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Recipe Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="recipeName"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="addIngredientButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QCheckBox" name="nutBox">
     <property name="text">
      <string>Nut Free</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTableView" name="ingredientsList"/>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="glutenBox">
     <property name="text">
      <string>Gluten Free</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="ingredientPrice"/>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="ingredientQuantity"/>
   </item>
   <item row="10" column="2">
    <widget class="QCheckBox" name="vegBox">
     <property name="text">
      <string>Vegetatian</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Dietary Restrictions</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QCheckBox" name="dairyBox">
     <property name="text">
      <string>Dairy Free</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ingredients</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="ingredientName">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Ingredient Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRecipe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRecipe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
